packages/fast-components-react-base/src/media/Media.props.ts

```diff
@@ -0,0 +1,48 @@
+import * as React from "react";
+
+/**
+ * Media HTML tags
+ */
+export enum MediaHTMLTags {
+    img = "img",
+    picture = "picture"
+}
+
+export interface IMediaProps {
+    /**
+     * The HTML alt attribute value is important for overall accessibility, providing a textual 
+     * alternative to non-text content 
+     */
+    alt: string;
+
+    /**
```

Comment:
Not sure this needs to be a first class prop. A user should be able to pass this via unhandled props and I don't know that there's a good reason to add as a first-class prop since it's not required for an image.